neonvm/runner/main.go

```diff
@@ -492,12 +494,12 @@ func main() {
 	// going to resize
 	if !vmSpec.Guest.RootDisk.Size.IsZero() {
 		if vmSpec.Guest.RootDisk.Size.Cmp(*imageSizeQuantity) == 1 {
-			log.Printf("resizing rootDisk from %s to %s", imageSizeQuantity.String(), vmSpec.Guest.RootDisk.Size.String())
+			logger.Info(fmt.Sprintf("resizing rootDisk from %s to %s", imageSizeQuantity.String(), vmSpec.Guest.RootDisk.Size.String()))
 			if err := execFg(QEMU_IMG_BIN, "resize", rootDiskPath, fmt.Sprintf("%d", vmSpec.Guest.RootDisk.Size.Value())); err != nil {
-				log.Fatal(err)
+				logger.Fatal("Failed to resize rootDisk", zap.Error(err))
 			}
 		} else {
-			log.Printf("rootDisk.size (%s) should be more than size in image (%s)", vmSpec.Guest.RootDisk.Size.String(), imageSizeQuantity.String())
+			logger.Info(fmt.Sprintf("rootDisk.size (%s) is greater than image size (%s)", vmSpec.Guest.RootDisk.Size.String(), imageSizeQuantity.String()))
```

Comment:
seems like a false statement. There we have to report that the size in the VM specification is smaller than the size in qcow2. We can only resize up, not down.

Reply:
Ah right, yeah - fixed.